src/Summoner/Config.hs

```diff
@@ -151,14 +154,15 @@ configT = Config
     <*> lastT (Toml.table preludeT) "prelude" .= cPrelude
     <*> textArr         "extensions"  .= cExtensions
     <*> textArr         "warnings"    .= cWarnings
+    <*> lastT sourceT   "stylish"     .= cStylish
   where
     lastT :: (Key -> BiToml a) -> Key -> BiToml (Last a)
     lastT = Toml.wrapper . Toml.maybeT
 
-    _GhcVer :: Prism AnyValue GhcVer
-    _GhcVer = Prism
-        { preview = \(AnyValue t) -> Toml.matchText t >>= parseGhcVer
-        , review = AnyValue . Toml.Text . showGhcVer
+    _GhcVer :: BiMap AnyValue GhcVer
+    _GhcVer = BiMap
+        { forward = \(AnyValue t) -> Toml.matchText t >>= parseGhcVer
+        , backward = Just . AnyValue . Toml.Text . showGhcVer
```

Comment:
There should be easier way to create such `BiMap` in `tomland`. It's not implemented currently but looks like we need it.